lib/index.js

```diff
@@ -62,6 +66,18 @@ function start(startingConfig) {
   };
 }
 
+function setFlushOnExit() {
+  function handleBeforeExit(exitCode) {
+    debug('flushing last beacons before exiting');
+    transmitter.handlePeriodicTasks()
+      .finally(() => {
```

Comment:
node10 syntax, might change to duplicate `then` and `catch` logic so we still support node8?